lib/components/Layout/Header.css

```diff
@@ -67,4 +66,8 @@
   display: none;
 }
 
+.branding {
+  padding-left: 24px;
```

Comment:
we should use var(--vds...) instead of '24px'

Reply:
fixed